docs/logging/05-01-logging.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Logging chart
+type: Configuration
+---
+
+To configure the Logging chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
+
+>**TIP:** To learn more about how to use overrides in Kyma, see the following documents:
+>* [Helm overrides for Kyma installation](/root/kyma/#configuration-helm-overrides-for-kyma-installation)
+>* [Top-level charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-top-level-charts-overrides)
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **persistence.enabled** | Specifies whether you store logs on a persistent volume instead of a volatile mounted volume. | `true` |
+| **persistence.size** | Defines the size of the persistent volume. | `10Gi` |
+| **config.auth_enabled** | Specifies the authentication mechanism you use to access the logging service. Set it to `false` to use Dex authentication or to `true` to use the basic HTTP authentication instead.  | `false` |
```

Comment:
would be much better if we say `use built in istio authentication/authorization` instead of saying `use Dex authentication`